test/integration/cmds/extension/snapshots/create.test.js.md

```diff
@@ -11,14 +11,16 @@ Generated by [AVA](https://ava.li).
     `contentful.js extension create␊
     ␊
     Options:␊
-      -h, --help     Show help                                             [boolean]␊
-      --id           Extension id                                           [string]␊
-      --name         Extension name                                         [string]␊
-      --space-id     Space id                                               [string]␊
-      --field-types  Field types                                             [array]␊
-      --descriptor   Path to an extension descriptor file                   [string]␊
-      --src          URL to extension bundle                                [string]␊
-      --srcdoc       Path to extension bundle                               [string]␊
-      --sidebar      Render the extension in the sidebar                   [boolean]␊
+      -h, --help                 Show help                                 [boolean]␊
+      --id                       Extension id                               [string]␊
+      --name                     Extension name                             [string]␊
+      --space-id                 Space id                                   [string]␊
+      --field-types              Field types                                 [array]␊
+      --descriptor               Path to an extension descriptor file       [string]␊
+      --src                      URL to extension bundle                    [string]␊
+      --srcdoc                   Path to extension bundle                   [string]␊
+      --sidebar                  Render the extension in the sidebar       [boolean]␊
+      --installation-parameters  JSON string of installation parameter values␊
```

Comment:
if you say this is a JSON string, then also do that in the above examples where its just a string